src/details/ArborX_AccessTraits.hpp

```diff
@@ -153,7 +153,7 @@ void check_valid_access_traits(PredicatesTag, Predicates const &)
                 "Invalid tag for the predicates");
 }
 
-template <typename Primitives>
+template <typename Primitives, bool ReturnAny = false>
 void check_valid_access_traits(PrimitivesTag, Primitives const &)
```

Comment:
This is hijacking `AccessTraits` to behave as `RangeTraits` within `BasicBoundingVolumeHierarchy`. Introducing `RangeTraits` is currently very hard, and would require massive code changes. I decided to simplify my life in this patch through this hijacking to make progress.